mesheryctl/pkg/utils/error.go

```diff
@@ -340,6 +343,30 @@ func formatError(msg string, cmd cmdType) string {
 	}
 }
 
+func ErrExtractFile(err error, msg string) error {
+	return errors.New(ErrExtractFileCode, errors.Alert,
+		[]string{"Failed to extract " + msg + " file"},
+		[]string{err.Error()},
+		[]string{"Error occurred while extracting the file"},
+		[]string{"Check the file path and ensure it is a valid " + msg + " file."})
```

Comment:
Is `msg` a filetype? If so: ok.

If not, this phrasing is wrong and will need updated.